Userland/Libraries/LibArchive/Tar.cpp

```diff
@@ -24,10 +24,12 @@ unsigned TarFileHeader::expected_checksum() const
     return checksum;
 }
 
-void TarFileHeader::calculate_checksum()
+ErrorOr<void> TarFileHeader::calculate_checksum()
 {
     memset(m_checksum, ' ', sizeof(m_checksum));
-    VERIFY(DeprecatedString::formatted("{:06o}", expected_checksum()).copy_characters_to_buffer(m_checksum, sizeof(m_checksum)));
+    bool copied_expected_checksum_to_internal_checksum = TRY(String::formatted("{:06o}", expected_checksum())).bytes_as_string_view().copy_characters_to_buffer(m_checksum, sizeof(m_checksum));
+    VERIFY(copied_expected_checksum_to_internal_checksum);
```

Comment:
That variable name sounds a bit excessive, `copy_successful` is much shorter and probably suffices as well considering the context.